play-json/js/src/main/scala/EnvReads.scala

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.api.libs.json
+
+trait EnvReads {
+  // No specific reader
```

Comment:
There is no instances of the typeclasses `Reads` and `Writes` specific to the ScalaJS backend